doc/sphinx-guides/source/api/native-api.rst

```diff
@@ -23,7 +23,7 @@ Create a Dataverse
 Generates a new dataverse under ``$id``. Expects a JSON content describing the dataverse, as in the example below.
 If ``$id`` is omitted, a root dataverse is created. ``$id`` can either be a dataverse id (long) or a dataverse alias (more robust). ::
 
-    POST http://$SERVER/api/dataverses/$id?key=$apiKey
+``curl -H "X-Dataverse-key:$API_TOKEN" http://$SERVER/api/dataverses/$id --upload-file sample.json``
```

Comment:
A few things (I'm happy to fix these if it's easier):

- We need to add -X POST back in.
- Let's change `http://$SERVER` to `$SERVER_URL`
- Let's use dataverse-complete.json to match the